src/cable/server.cr

```diff
@@ -182,8 +171,10 @@ module Cable
       spawn(name: "Cable::Server - process_subscribed_messages") do
         while received = fiber_channel.receive
           channel, message = received
-          if channel.includes?("cable_internal")
-            server.send_to_internal_channels(channel, message)
+          if channel.starts_with?("cable_internal")
+            identifier = channel.split('/').last
+            connection_identifier = server.connections.keys.find!(&.starts_with?(identifier))
+            server.send_to_internal_connections(connection_identifier, message)
```

Comment:
The `channel` will look like `cable_internal/abc123`, but we actually need `abc123-random-uuid` to know which connection to disconnect